### ISSUE TITLE
Acute and grave accents are swapped in Laila

### ISSUE THREAD

**Issue by laerm0** [google/fonts#1155]
All weights of Laila have the acute and grave characters swapped. The outlines need to be swapped (or the glyphs need to be reencoded).

https://code.google.com/archive/p/googlefontdirectory/issues/382

**Comment by egcmi** [comment]
no progress? still affected by this issue 3 years later

**Comment by thlinard** [comment]
Indeed! https://fonts.google.com/specimen/Laila?preview.text=%C3%81%C3%89%C3%8D%C3%93%C3%9A%C3%A1%C3%A9%C3%AD%C3%B3%C3%BA%C3%80%C3%88%C3%8C%C3%92%C3%99%C3%A0%C3%A8%C3%AC%C3%B2%C3%B9&preview.text_type=custom

**Comment by jqu-odoo** [comment]
This is still true today.

**Comment by MyBookings** [comment]
Hi, we are experiencing the same issue. Any update on this one? Thanks!
